xep-0420.xml

```diff
@@ -395,7 +401,7 @@
 </section1>
 
 <section1 topic='XMPP Registrar Considerations' anchor='registrar'>
-  <p>TODO: Maybe the Registrar should handle a blacklist of elements that are allowed as child elements of the &content; element?</p>
+  <p>TODO: Maybe the Registrar should handle a list of elements that are allowed as child elements of the &content; element?</p>
```

Comment:
I think this flips the logic doesn't it?

Reply:
It does indeed, thanks for the catch! Pushed a fix.